deploy/olm-catalog/windows-machine-config-operator/manifests/windows-exporter-role.yaml

```diff
@@ -0,0 +1,15 @@
+# Roles needed by prometheus to scrape windows metrics endpoint
```

Comment:
Is there no way to define this inside the CSV? Moreover this is now missing from the `deploy` folder which implies, we have to use OLM to deploy the operator and can't do a direct deployment.

Reply:
Tested by appending `role` and `role-binding` to existing files and it did not work.

However, when moving to operator-sdk v1.0, we could add them in the [`config/rbac` ](https://sdk.operatorframework.io/docs/building-operators/golang/migration/)folder.